### PR TITLE
range filter

### PR DESCRIPTION
**Pull request by smknstd**

### CODE REVIEW

**Comment by smknstd** [comment]
todo:
- example in saturn
- unit test
- update docs

File: src/EntityList/EntityListFilter.php [hunk 1]
```diff
@@ -3,13 +3,18 @@
...
 use Code16\Sharp\Utils\Filters\ListFilter;
+use Code16\Sharp\Utils\Filters\ListRangeFilter;
 use Code16\Sharp\Utils\Filters\ListMultipleFilter;
 use Code16\Sharp\Utils\Filters\ListRequiredFilter;
 
 interface EntityListFilter extends ListFilter
 {
 }
 
+interface EntityRangeFilter extends ListRangeFilter
```
Comment (dvlpp) [inline src/EntityList/EntityListFilter.php]: Should be Entity**List**RangeFilter

File: saturn/app/Sharp/Filters/PassengerBirthdateFilter.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+use Code16\Sharp\EntityList\EntityListDateRangeFilter;
+
+class PassengerBirthdateFilter implements EntityListDateRangeFilter
+{
+
+    /**
+     * @return array
+     */
+    public function values()
```
Comment (dvlpp) [inline saturn/app/Sharp/Filters/PassengerBirthdateFilter.php]: Remove this

File: src/Utils/Filters/DateRangeFilter.php [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+interface DateRangeRequiredFilter extends DateRangeFilter
+{
+    /**
+     * @return array
+     *
+     * awaited format:
+     *
+     *    [
+     *       "start" => 20180101,
```
Comment (dvlpp) [inline src/Utils/Filters/DateRangeFilter.php]: Use Carbon dates

**Comment by dvlpp** [comment]
This PR partially address #24 (Date Range filters)

**Pull request merged**